src/vault.ts

```diff
@@ -5,7 +5,9 @@ const pexec = util.promisify(oexec);
 
 const slice = process.argv[2] === '.' ? 3 : 2
 const ARGS = process.argv.slice(slice);
-const OPTIONS = require('simple-argv')
+const yargs = require('yargs/yargs')
```

Comment:
@cfloress Could it be standardized to have the constants names in the same way (common convention is uppercase)

Reply:
Sorry,  I don't follow. The `OPTIONS` constant is already in upper cases and has been moved 2 lines below; in case you are referring to yargs, I see it returns an instance when required ([ref](https://github.com/yargs/yargs/blob/main/index.mjs#L8)) so it is ok to be in lower cases.

Still, I may be overlooking at something so correct me if I'm wrong there.